$NetBSD: patch-ac,v 1.11 2016/03/07 16:27:47 jperkin Exp $

Fix handling of installation directories.
Install static libs with the right permissions.
Don't use non-portable -z discard-unused on SunOS.

--- Makefile.rules.orig	2015-03-03 20:54:29.000000000 +0000
+++ Makefile.rules
@@ -641,7 +641,7 @@ ifndef NO_DEAD_STRIP
   ifeq ($(HOST_OS),Darwin)
     LD.Flags += -Wl,-dead_strip
   else
-    ifeq ($(HOST_OS),SunOS)
+    ifeq ($(HOST_OS),notSunOS)
       LD.Flags += -Wl,-z -Wl,discard-unused=sections
     else
       ifneq ($(HOST_OS), $(filter $(HOST_OS), Cygwin MingW))
@@ -847,9 +847,6 @@ endif
 # in the file so they get built before dependencies
 #---------------------------------------------------------
 
-$(DESTDIR)$(PROJ_bindir) $(DESTDIR)$(PROJ_libdir) $(DESTDIR)$(PROJ_includedir) $(DESTDIR)$(PROJ_etcdir)::
-	$(Verb) $(MKDIR) $@
-
 # To create other directories, as needed, and timestamp their creation
 %/.dir:
 	$(Verb) $(MKDIR) $* > /dev/null
@@ -993,7 +990,9 @@ install-local::
 uninstall-local::
 	$(Echo) UnInstall circumvented with NO_INSTALL
 else
-install-local:: $(DESTDIR)$(PROJ_etcdir) $(CONFIG_FILES)
+PROJ_etcdir=${PREFIX}/share/examples/llvm
+install-local:: $(CONFIG_FILES)
+	$(MKDIR) ${PROJ_etcdir}
 	$(Echo) Installing Configuration Files To $(DESTDIR)$(PROJ_etcdir)
 	$(Verb)for file in $(CONFIG_FILES); do \
           if test -f $(PROJ_OBJ_DIR)/$${file} ; then \
@@ -1305,7 +1304,7 @@ install-local:: $(DestArchiveLib)
 $(DestArchiveLib): $(LibName.A) $(DESTDIR)$(PROJ_libdir)
 	$(Echo) Installing $(BuildMode) Archive Library $(DestArchiveLib)
 	$(Verb) $(MKDIR) $(DESTDIR)$(PROJ_libdir)
-	$(Verb) $(INSTALL) $(LibName.A) $(DestArchiveLib)
+	$(Verb) $(DataInstall) $(LibName.A) $(DestArchiveLib)
 
 uninstall-local::
 	$(Echo) Uninstalling $(BuildMode) Archive Library $(DestArchiveLib)
